.mocharc.js

```diff
@@ -10,4 +10,13 @@ module.exports = {
   recursive: true,
   reporter: 'dot',
   require: [require.resolve('./test/utils/setupBabel'), require.resolve('./test/utils/setupJSDOM')],
+  'watch-ignore': [
+    // default
+    '.git',
+    // node_modules can be nested with workspaces
+    '**/node_modules/**',
```

Comment:
mocha includes `'node_modules'` which does not cover nested n_m. Nested n_m exist when using workspaces.